contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/plugins/image-upload/image-upload.css

```diff
@@ -1,4 +1,4 @@
-.tui-toolbar-btn-image-upload {
+.tui-toolbar-btn-image-upload.tui-toolbar-icons {
```

Comment:
This was necessary to make sure the image upload button icon showed up.  I'm not exactly sure why the formula button's icon worked but the image upload button's icon didn't work, but I suspect it had to do with style ordering in webpack, since from what I could tell, there are zero differences in how `.tui-toolbar-btn-image-upload` and `.tui-toolbar-btn-formula` are defined and used.